sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -1104,3 +1120,314 @@ def test_global_forecasting_no_X(self, estimator_instance):
 
         # check consistency of forecast hierarchy with training data
         assert set(y_test.index.names).issubset(y_pred.index.names)
+
+    def _check_predict_intervals(self, pred_ints, y_test, fh, coverage):
```

Comment:
Please check again if you can reuse functionality from _check_predict. I suppose that the for example the mtype checks etc. are implemented there too